app/domain/logs.rb

```diff
@@ -614,6 +614,21 @@ module AuthnJwt
         code: "CONJ00121D"
       )
 
+      FetchingMandatoryClaims = ::Util::TrackableLogMessageClass.new(
+        msg: "Fetching mandatory claims...",
+        code: "CONJ00122D"
+      )
+
+      NotConfiguredMandatoryClaims = ::Util::TrackableLogMessageClass.new(
+        msg: "Mandatory claims is not configured",
```

Comment:
I think mandatory claims **are** not configured
unless you are saying Mandatory claims **variable** is not configured